framework/helpers/BaseHtml.php

```diff
@@ -819,6 +819,7 @@ protected static function booleanInput($type, $name, $checked = false, $options
      *   Defaults to false.
      * - encode: bool, whether to encode option prompt and option value characters.
      *   Defaults to `true`. This option is available since 2.0.3.
+     * - strict: boolean, if `$selection` is an array and this value is true a strict comparison will be performed on `$items` keys. Defaults to false.
```

Comment:
Need to add "since 2.0.37"

Reply:
I don't think `booleanInput` is effected by this.

Reply:
Never mind, it's showing above as `booleanInput` but I've added since to the `dropDownList` method.